internal/sync/sync.go

```diff
@@ -71,12 +71,17 @@ func NewSynchronizer(cfg *config.Sync, feederClient *feeder.Client, syncManager
 	synchro := new(Synchronizer)
 	synchro.logger = Logger.Named("Sync Service")
 	synchro.feeder = feederClient
```

Comment:
Let's use a composite literal here instead. See [section in Effective Go](https://go.dev/doc/effective_go#composite_literals).

```suggestion
	synchro := &Synchronizer{
		logger: Logger.Named("Sync Service"),
		feeder: feederClient,
	}
```